test/utils.ts

```diff
@@ -28,7 +28,7 @@ export function getSVGParent(): AnySelection {
 
 const PIXEL_CLOSETO_REQUIREMENT = 2;
 
-export function assertBBoxInclusion(outerEl: AnySelection, innerEl: AnySelection) {
+export function assertBBoxInclusion(outerEl: d3Selection<any>, innerEl: d3Selection<any>) {
   const outerBox = (outerEl.node() as Element).getBoundingClientRect();
```

Comment:
outerEl should be `d3Selection<Element>` or something if we're gonna cast it like this. Same with all the other places we do casts on d3Selection's methods